src/app/components/profile/profile.html

```diff
@@ -83,7 +83,7 @@
 
     <!-- PRIVATE INFO -->
 
-    <div class="col-md-8 col-md-offset-2 col-lg-6 col-lg-offset-3 col-sm-12" ng-hide="true">
+    <div class="col-md-8 col-md-offset-2 col-lg-6 col-lg-offset-3 col-sm-12" <!-- ng-hide="true" -->>
```

Comment:
hmm, why? :) havnt pulled this request yet but wont this show the private info? (which should be disabled for now)